packages/renderer/src/lib/deployments/DeploymentColumnNamespace.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
```

Comment:
I'm wondering why we don't have a generic NamespaceColumn as it's likely that we will have a lot of Kubernetes objects that are namespace aware